### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]
Looks good to me. Did you try running `ansible-test sanity --docker -v` on this?

**Comment by rndmh3ro** [comment]
Thanks @felixfontein, I just did that and found some more files to change.

There's only one error remaining:

```
ERROR: plugins/modules/icinga_deploy.py:0:0: collection-deprecated-module: Module is marked for removal in version 2.0.0 of this collection when the current version is 2.0.1
```

If I change removal_version to 2.0.1 in all files, I get that error:

```
ERROR: plugins/modules/icinga_deploy_info.py:0:0: removal-version-must-be-major: DOCUMENTATION.deprecated: removed_in ('2.0.1') must be a major release, not a minor or patch release (see specification at ht
tps://semver.org/) for dictionary value @ data['deprecated']. Got {'why': 'moved collection to new organisation telekom_mms', 'removed_in': '2.0.1', 'alternative': 'telekom_mms.icinga_director.icinga_deploy
_info', 'removed_from_collection': 't_systems_mms.icinga_director'}
```

Should we ignore this or do a final 3.0.0 release?

File: meta/runtime.yml [hunk 1]
```diff
@@ -43,3 +43,212 @@ action_groups:
...
     - icinga_zone
     - icinga_zone_info
+
+plugin_routing:
+  inventory:
+    icinga_director_inventory:
+      redirect: telekom_mms.icinga_director.icinga_director_inventory
+      deprecation:
+        warning_text: Use telekom_mms.icinga_director.icinga_director_inventory instead
+        removal_version: 2.0.0
```
Comment (felixfontein) [inline meta/runtime.yml]: This is the version when the redirect will be removed:
```suggestion
        removal_version: 3.0.0
```
Same below.

Comment (rndmh3ro) [reply meta/runtime.yml]: Thanks, I changed it to 3.0.0 everywhere.

**Review by felixfontein — APPROVED** [review]

**Pull request merged**